zh_CN.ts

```diff
@@ -297,11 +297,11 @@ export const ZH = {
     Right: "右",
     Auto: "自动",
     HighlightBuildings: "高亮建筑",
-    HighlightTurnedOff: "...被关闭的",
-    HighlightStockpileModeOn: "...开启库存模式的",
-    HighlightWarehouseOnly: "...只供应仓库的",
-    HighlightProductionCycleNotDefault: "...生产周期 > 1的",
-    HighlightMaxInputDistanceNotDefault: "...最大输入距离 < ∞的",
+    HighlightTurnedOff: "被关闭的...",
+    HighlightStockpileModeOn: "开启库存模式的...",
+    HighlightWarehouseOnly: "只供应仓库的...",
+    HighlightProductionCycleNotDefault: "生产周期 > 1的...",
+    HighlightMaxInputDistanceNotDefault: "最大输入距离 < ∞的...",
     HighlightNotMakingProfit: "...不盈利的",
```

Comment:
你忘了这个
You forgot this
```suggestion
    HighlightNotMakingProfit: "不盈利的...",
```